docker-compose.yml

```diff
@@ -1,307 +1,116 @@
+      - LOG_LEVEL=DEBUG
+      - FHIRSTORE_USER=arkhn
+      - FHIRSTORE_PASSWORD=5gBwjTYR4DHSbzaUwVNX43DbTn
+      - REDIS_REFERENCES_HOST=river-redis
+      - REDIS_REFERENCES_PORT=6379
+      - REDIS_REFERENCES_DB=0
+      - REDIS_MAPPINGS_HOST=river-redis
+      - REDIS_MAPPINGS_PORT=6379
+      - REDIS_MAPPINGS_DB=1
+      - REDIS_COUNTER_HOST=river-redis
+      - REDIS_COUNTER_PORT=6379
+      - REDIS_COUNTER_DB=2
+
+  control-api:
+    image: river
+    build: .
+    restart: on-failure
+    environment:
+      - ENV=dev
+      - LOG_LEVEL=DEBUG
+      - DEBUG=True
+      - SECRET_KEY=whatever
 
-  # Uncomment if you want monitoring
-  # prometheus:
-  #   container_name: prometheus
-  #   image: prom/prometheus:latest
-  #   volumes:
-  #     - ./prometheus.yaml:/etc/prometheus/prometheus.yml
-  #   ports:
-  #     - "9090:9090"
-  #   networks:
-  #     - river-network
-  # grafana:
-  #   container_name: grafana
-  #   image: grafana/grafana:latest
-  #   ports:
-  #     - 3010:3000
-  #   volumes:
-  #     - ./grafana/config.ini:/etc/grafana/config.ini
-  #     - ./grafana/provisioning:/etc/grafana/provisioning
-  #     - ./grafana/dashboards:/var/lib/grafana/dashboards
-  #   networks:
-  #     - river-network
+  api:
+    build: api
+    restart: on-failure
+    depends_on:
+      - kafka
+    environment:
+      - PORT=3000
+      - KAFKA_BOOTSTRAP_SERVERS=kafka:9092
+      - CONTROL_API_URL=http://control-api:8000
+      - PYROG_API_URL=http://pyrog-server:1000
+      - FHIR_API_URL=http://fhir-api:2000
+      - REDIS_MAPPINGS_HOST=river-redis
+      - REDIS_MAPPINGS_PORT=6379
+      - REDIS_MAPPINGS_DB=1
+      - REDIS_COUNTER_HOST=river-redis
+      - REDIS_COUNTER_PORT=6379
+      - REDIS_COUNTER_DB=2
 
-  fluentd:
-    image: arkhn/fluentd:latest
-    container_name: fluentd
-    volumes:
-      - ./fluentd/conf:/fluentd/etc
-    ports:
-      - 127.0.0.1:24224:24224
-      - "24224:24224/udp"
-    networks:
-      - river-network
+  zookeeper:
+    image: zookeeper:3.4.10
+    restart: on-failure
 
-  elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.7.0
-    container_name: elasticsearch
-    volumes:
-      - ./elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
-    expose:
-      - 9200
+  kafka:
+    image: wurstmeister/kafka:2.13-2.6.0
+    restart: on-failure
+    # This port mapping is required by the containerized kafka
     ports:
-      - "9200:9200"
-    networks:
-      - river-network
-
-  kibana:
-    image: docker.elastic.co/kibana/kibana:7.7.0
-    container_name: kibana
+      - "9092:9092"
     depends_on:
-      - elasticsearch
-    ports:
-      - "5601:5601"
-    networks:
-      - river-network
+      - zookeeper
+    environment:
+      - KAFKA_LISTENERS=PLAINTEXT://:9092
+      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092
+      - KAFKA_DELETE_TOPIC_ENABLE=true
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
+      # - KAFKA_HEAP_OPTS=-Xmx256M -Xms128M
```

Comment:
why the comment ?

Reply:
Not required nor recommanded in production. Comment added.